package.json

```diff
@@ -116,7 +116,7 @@
     "eslint-plugin-react": "^7.24.0",
     "file-loader": "^6.2.0",
     "husky": "^4.3.8",
-    "jest": "^27.3.1",
+    "jest": "^27.5.1",
```

Comment:
These should not be added.

Reply:
@3ru 
I tried git commands but I did not managed to edit my previous commit with package.json, and ended up commiting one more change. how do I unstage package.json from [b9e4bef] this commit , and is it possible to delete the unwanted commits after that?

Reply:
@Aryan1982 
The maintainer can use squash, etc., to summarize when merging, so there is no problem with commits being added.

For now, only `client/components/Nav.jsx` has been added, and it will fail the test again.

- update
`jest --updateSnapshot`
- add
`git add client/components/__snapshots__/Nav.unit.test.jsx.snap`

After that, commit and push.

---
It is also a good idea to try `npm run test` in your own environment to see if the test fails.

If the test passes as follows, you will not fail in CI.

```bash
Test Suites: 15 passed, 15 total
Tests:       109 passed, 109 total
Snapshots:   2 passed, 2 total
```

Reply:
@3ru 
Thank you! I have updated the snapshot, and going to add and commit changes

Reply:
@3ru 
got the following
` PASS   client  client/components/Nav.unit.test.jsx (10.815 s)
 PASS   client  client/modules/IDE/components/FileNode.unit.test.jsx (9.3 s)
 PASS   server  server/controllers/user.controller/__tests__/apiKey.test.js (5.711 s)
 PASS   server  server/controllers/project.controller/__test__/deleteProject.test.js (7.267 s)
 PASS   server  server/controllers/project.controller/__test__/getProjectsForUser.test.js (9.191 s)
 PASS   server  server/domain-objects/__test__/Project.test.js (7.213 s)
 PASS   server  server/models/__test__/project.test.js
 PASS   client  client/modules/IDE/actions/projects.unit.test.js (5.441 s)
 PASS   client  client/utils/getConfig.test.js

Test Suites: 1 failed, 14 passed, 15 total
Tests:       1 failed, 108 passed, 109 total
Snapshots:   2 passed, 2 total`

Reply:
snapshots was failing before, after updating..its fine now, 
what about the other tests?

Reply:
@Aryan1982 
The `Nav.unit.test.jsx` test seems to pass, so I think this PR will be OK if you push the snapshot and commit.

That test is failing is another issue.